ecommers/lib/ui/pages/profile_edit_page.dart

```diff
@@ -0,0 +1,139 @@
+import 'package:ecommers/core/models/edit_profile_model.dart';
+import 'package:ecommers/core/provider_models/index.dart';
+import 'package:ecommers/shared/dependency_service.dart';
+import 'package:ecommers/ui/decorations/dimens/insets.dart';
+import 'package:ecommers/ui/decorations/index.dart';
+import 'package:ecommers/ui/pages/busy_page.dart';
+import 'package:ecommers/ui/widgets/button/primary_button_widget.dart';
+import 'package:ecommers/ui/widgets/index.dart';
+import 'package:ecommers/ui/widgets/profile/profile_image.dart';
+import 'package:flutter/material.dart' hide BackButton;
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:provider/provider.dart';
+
+class ProfileEditPage extends StatelessWidget {
+  static const EdgeInsets _listContainerMargin =
+      EdgeInsets.symmetric(horizontal: Insets.x5);
```

Comment:
We decide to don't create consts for margins. You don't reuse  value

Reply:
Moved into body